apps/mobile/src/features/balances/bitcoin/bitcoin-balance.tsx

```diff
@@ -13,17 +12,22 @@ import { TokenBalance } from '../token-balance';
 interface BitcoinTokenBalanceProps {
   availableBalance: Money;
   fiatBalance: Money;
+  onPress?(): void;
 }
-export function BitcoinTokenBalance({ availableBalance, fiatBalance }: BitcoinTokenBalanceProps) {
+export function BitcoinTokenBalance({
+  availableBalance,
+  fiatBalance,
+  onPress,
+}: BitcoinTokenBalanceProps) {
   return (
     <TokenBalance
       ticker="BTC"
       icon={<BtcAvatarIcon />}
       tokenName={t`Bitcoin`}
-      chain={t`Bitcoin blockchain`}
+      chain={t`Layer 1`}
```

Comment:
Do you need to show `Layer 1` ? 

I think we can actually even remove this field as I have used `showChain={false}` to not show it in `<TokenBalance` 

In the design we are supposed to show `Bitcoin blockchain` / `Stacks blockchain` for all non BTC / STX tokens

Reply:
I did confirm in Slack that designs have changed to `Layer 1`: https://trustmachines.slack.com/archives/C05LRS7G44R/p1731024428631009